### DISCUSSION TITLE
서버 아키텍처 질문 드립니다 😀

### DISCUSSION THREAD

**Discussion by wlswo** [MoCo-v2/moco-v2-backend#4] [Q&A] [answered]
기존 프로젝트는 레이어드 아키텍처로 구축해놓은 상태입니다.

헥사고날 아키텍처로 변경하는것이 현재 도메인 구성에 좋을지 고민입니다.

**Comment by HJ97Kim** [comment, marked as answer]
프로젝트 구조를 살펴보았습니다.
레이어드 아키텍처 그래도 진행하는게 좋아보입니다. 저희 프로젝트 규모로 보았을때 적합한 아키텍처로 생각됩니다.

**Comment by HJ97Kim** [comment]
마이그레이션을 고민중이시라면 추후 프로젝트 1차 프로토타입이 개발된후 진행하는 것이 좋아보입니다
